test/fw/ptl/utils/jobs/eatcpu.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/python
+import signal
+import time
+import sys
+
+# Usage: eatcpu.py [seconds]
+# Eats CPU resources for the time specified by user
+
+x = 0
+
+
+def receive_alarm(signum, stack):
+    sys.exit()
+
+signal.signal(signal.SIGALRM, receive_alarm)
+
+if (len(sys.argv) > 1):
+    input_time = sys.argv[1]
+    print 'Terminating after %s seconds' % input_time
+    signal.alarm(int(input_time))
+else:
+    print 'Running indefinitely'
+
+while 1:
+    x += 1
+    time.sleep(1)
```

Comment:
With this sleep here, this script won't really eat cputime.  It will do one math operation and then sleep for a second.  You will spend much more time sleeping than doing math operations.  The eatcpu.pl worked since it didn't sleep.  It did a while 1: x++.  This would do endless math operations.  Using an alarm will have a side effect I don't know if you want.  If you suspend this job for a length of time, the alarm doesn't suspend with it.  It could very well stop immediately when the job is continued.  Offhand, I can't think of a good way to fix that.

Reply:
Ah! Yes.  I will remove the sleep. And, on the second comment on using alarm, since the current tests don't have any job that will be suspended, I guess, I can add that capability later in a different PR. Any thoughts?

Reply:
Yeah, that is fine.  I just thought I'd mention it since I noticed it.  Even if we wanted to fix it, I don't have any good suggestions for you.  Let's just leave it alone for now.